samza-core/src/main/java/org/apache/samza/zk/ZkBarrierForVersionUpgrade.java

```diff
@@ -152,6 +171,11 @@ public void handleDataChange(String dataPath, Object data)
       if (done.equals(BARRIER_DONE)) {
```

Comment:
Is there any documentation on the allowed values for this ZkNode? What do each of these values mean?